### ISSUE TITLE
Videos in webview not loading

### ISSUE THREAD

**Issue by levrik** [microsoft/vscode#173260]
Type: <b>Bug</b>

Since VS Code 1.75.0 videos aren't loading in webviews anymore. For example extension changelogs or also VS Code's release notes. 

![image](https://user-images.githubusercontent.com/9491603/216548778-66037f0c-d3ca-4898-9a07-ee1414c381eb.png)

VS Code version: Code 1.75.0 (Universal) (e2816fe719a4026ffa1ee0189dc89bdfdbafb164, 2023-02-01T15:24:42.903Z)
OS version: Darwin arm64 22.2.0
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Pro (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|2, 3, 3|
|Memory (System)|16.00GB (0.09GB free)|
|Process Argv|--crash-reporter-id f6ebf509-193f-4637-bdee-0d6be9cade92|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyone:30548225
vscccc:30610679
pyindex848:30577860
nodejswelcome1cf:30587006
2e4cg342:30602488
f6dab269:30613381
pythonsymbol12:30657548
a9j8j154:30646983
vscsb:30628654

```

</details>

**Comment by levrik** [comment]
Mhh, I'm sorry. I don't know why but suddenly videos are loading except for the one on the GitHub extension's changelog. Unsure if it's an issue with VS Code or the specific changelog.

**Comment by levrik** [comment]
Maybe it's also just happening on specific videos?
Just found a few videos in VS Code's release notes not loading again. Took a screenshot this time.

![image](https://user-images.githubusercontent.com/9491603/216590107-53f1229b-5688-42e3-8363-e9c4ee13c6ab.png)

**Comment by mjbvz** [comment]
I'm not able to reproduce this in the release notes:

![Screenshot 2023-02-06 at 2 00 08 PM](https://user-images.githubusercontent.com/12821956/217097449-3791b7fb-ad41-4fd2-ab96-c08d5839f2df.png)

Are you still seeing this? Do the videos eventually load if you leave the release notes open for a moment?

**Comment by levrik** [comment]
@mjbvz It behaved very flaky. Sometimes all videos loaded, sometimes just a few, sometimes none. If they didn't load they also didn't load eventually. Re-opening the release notes didn't help, restarting/reloading VS Code helped resetting the broken state but didn't guarantee that all videos load. I couldn't reproduce it by visiting the release notes on VS Code's homepage.
The issue doesn't seem to happen anymore so maybe it was a server-side failure. I didn't think of checking the dev tools when it happened so I don't know what responses were received there. But I noticed now that URLs are different between release notes in VS Code and the homepage, this would at least explain why the homepage worked for me.
Closing this issue now. If it happens again, I'll inspect the requests and post an update here.
